content/Hardware Support/Core Boards/Whats-the-difference-between-UNO-R3-and-UNO-R4-boards.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "What's the difference between UNO R3 and UNO R4 boards?"
+---
+
+Learn about the differences between UNO R3 and the new UNO R4 WiFi and UNO R4 Minima boards.
+
+---
+
+## Select differences
```

Comment:
Although they have similar meanings, I would suggest using "Selected" or "Main" instead of "Select" (As this can be confusing and mislead the reader to think that they have to select something or follow a tutorial).